dbt_airflow_manifest_parser/operator.py

```diff
@@ -67,3 +68,7 @@ def _create(self, args, name):
             is_delete_operator_pod=self.kubernetes_execution_parameters.is_delete_operator_pod,
             hostnetwork=False,
         )
+
+
+class EphemeralOperator(DummyOperator):
+    ui_color = "#F3E4F7"
```

Comment:
Good idea, I forget about this feature. We could use different colors for different materializations maybe.